tdrs-backend/tdpservice/parsers/fields.py

```diff
@@ -53,7 +53,7 @@ def parse_value(self, line):
                     value = int(value)
                     return value
                 except ValueError:
-                    logger.error(f"Error parsing field value: {value} to integer.")
+                    logger.error(f"Error parsing field {self.name} value to integer.")
```

Comment:
could the call to `transform_func` in `parse_value` from the `TranformField` below also throw an exception?

```python
value = super().parse_value(line)
return self.transform_func(value, **self.kwargs)
```

Reply:
yes, for uncaught exceptions, there is a risk

Reply:
since transform fields are really only used for SSN, i think it's worth putting some exception handling so we're not risking a leak

Reply:
The problem I have with change is, it might impact the upper exception handling. It needs further investigation

Reply:
Did a couple of tests, and confirmed it is safe. See below the changes